app/views/article/record/_html_fulltext.erb

```diff
@@ -0,0 +1,13 @@
+<% doc_presenter = presenter(document) -%>
+<button class="section-container-heading fulltext-toggle-bar" id="fulltextToggleBar" data-toggle="collapse" href="#toggleFulltext" aria-expanded="false" aria-controls="toggleFulltext">
+  <h2>Full text</h2>
+</button>
+<div class='section-body collapse' id="toggleFulltext">
+<% fields.each do |field_name| -%>
+  <% field_name = field_name.to_s -%>
+  <% value = doc_presenter.field_value field_name -%>
+  <% if value.present? %>
+    <dd class="blacklight-<%= field_name.parameterize %>"><%= value %></dd>
```

Comment:
It looks like this is putting the content in a `dd` w/o any of the other markup (parent `dl` and an accompanying `dt`).

Check out the [Permitted parents section](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd) of this page.

I think we can just wrap this in a `div` and call it a day.